core-cube/src/main/java/org/apache/kylin/cube/model/CubeDesc.java

```diff
@@ -1375,12 +1376,37 @@ public void setParentForward(int parentForward) {
                     result.remove(dictDesc.getColumnRef());
                     result.add(dictDesc.getResuseColumnRef());
                 }
+
+                //tiretree global domain dic
+                if (Objects.isNull(dictDesc.getResuseColumnRef()) && Objects.nonNull(dictDesc.getReuseColumn())) {
+                    logger.info("tiretree global domain dic : column {} use tiretree global domain dic, reuse column {} ", dictDesc.getColumnRef(), dictDesc.getReuseColumn());
```

Comment:
I think it's better to put this into the scope of the previous conditional expression "if (dictDesc.getResuseColumnRef() != null) "

Reply:
Thanks for coding review .The previous conditional expression use for reuse col of the same cube (Existing functions of kylin), and dictDesc.getResuseColumnRef() is not null, but the new global domain dic's dictDesc.getResuseColumnRef() is null . After the expression is satisfied, the content to be processed is different too， the global domain dic only result.remove(dictDesc.getColumnRef()) no result.add(dictDesc.getResuseColumnRef()).